ui/app/pages/swaps/build-quote/build-quote.js

```diff
@@ -416,11 +418,13 @@ export default function BuildQuote({
                       <a
                         className="build-quote__token-etherscan-link build-quote__underline"
                         key="build-quote-etherscan-link"
-                        href={`https://etherscan.io/token/${selectedToToken.address}`}
+                        href={`${rpcPrefs.blockExplorerUrl}/token/${selectedToToken.address}`}
```

Comment:
Both this URL and the one on like 457 below will be incorrect for some URLs (e.g. anything with a query string, hash, API key, etc.). We should use our `@metamask/etherscan-link` library to generate these instead.

Reply:
Done in 7a5b0dd92